config/locales/en.json

```diff
@@ -504,11 +504,18 @@
 	"dashboard.overall-health": "Patient's Overall Health",
 	"dashboard.expedited-processing": "Expedited processing for terminal and grave conditions",
 	"conditions.incomplete": "Not Completed",
+	"invite.expires": "This code will expire on %s",
+	"invite.code_loss_instruction": "If you lose your code, or it expires, fill out this form again to generate another code",
 	"invite.title": "Form submitted",
-	"invite.give_code": "To finish your application, give this code and link to your doctor or nurse practioner:",
+	"invite.give_code": "To finish your medical report, give this code to your doctor or nurse practitioner.",
+	"invite.give_code_details": "They will use this code to fill out their part of the medical report form, and send it to us at Service Canada.",
 	"invite.medical_professional_instruction": "Your doctor or nurse practioner will use this unique code to fill out a report about your medical condition(s), and how they impact your ability to work.",
 	"invite.print_button": "Print this code",
 	"invite.url": "https://cppd-medical-report-stage.herokuapp.com/en/doctor",
+	"invite.for_medical_professionals": "For health professionals:",
```

Comment:
Can we put this in italics and maybe add a little more whitespace between this line and the box aimed at the applicant?
I want there to be more of gap between them to help emphasize the top bit is for the applicant, and this chunk of steps is for someone else.

Reply:
done